ui/core/components/detailed_results/timeline.tsx

```diff
@@ -866,6 +866,11 @@ export class Timeline extends ResultComponent {
 				});
 		});
 
+		// If there are any auras that correspond to this cast, visualize them in the same row.
+		aurasById
+			.filter(auraUptimeLogs => auraUptimeLogs[0].actionId!.equalsIgnoringTag(actionId))
+			.forEach(auraUptimeLogs => this.applyAuraUptimeLogsToRow(auraUptimeLogs, rowElem));
```

Comment:
Added this back after removing it recently because it really helps with seeing dot duration

Reply:
I was actually gonna ask why was this broken😅
Was not aware you removed it on purpose and i think it should stay as well as you say it helps a lot with durations visibility